gestion_base_adresse/test/test_execute_functions.py

```diff
@@ -200,3 +200,74 @@ def test_calcul_num_metrique(self):
             "from adresse.calcul_num_metrique(ST_geomfromtext('POINT(428252 6921965)', 2154))"
         )
         self.cursor.execute(sql)
+
+    def test_inversion_calcul_metrique(self):
+        # test avant puis apres Inversion de la voie et sens de numérotation
+        # (comme si l'on clique sur le bouton dans la popup)
```

Comment:
```suggestion
        """ Test avant puis après inversion de la voie et sens de numérotation
        (comme si l'on clique sur le bouton dans la popup) """
```
En utilisant une docstring et non pas un commentaire, cela permet de donner un titre au test (la première ligne d'une docstring). Celle elle qui s'affiche dans travis ou dans la console par exemple.